templates/post.html

```diff
@@ -43,6 +43,18 @@
 				{% endif %}
 				<span class="dot">&bull;</span>
 				<span class="created" title="{{ post.created }}">{{ post.rel_time }}</span>
+				{% if !post.awards.is_empty() %}
+				<span class="dot">&bull;</span>
+				<span class="awards">
+					{% for award in post.awards.clone() %}
+					<span class="tooltip">
+						<img alt="{{ award.icon_url }}" src="{{ award.icon_url }}" class="awards_image"/>
+						<span class="tooltiptext">{{award.name}}</span>
```

Comment:
Aside from style, was there a reason for choosing a custom tooltip over the built-in [browser tooltip](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title)?

Reply:
Not really, it is just for aesthetic purposes.